lib/utils/application_util.ex

```diff
@@ -25,4 +25,15 @@ defmodule EaRestaurantDataLoader.Lib.Utils.ApplicationUtil do
   def decode_json(json_data) do
     Poison.decode(json_data)
   end
+
+  def get_string_bson_object_id(bson_object_id) do
```

Comment:
You can remove this function, there is no need to add it.

Reply:
Done.